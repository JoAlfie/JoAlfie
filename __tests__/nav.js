import React from "react"
import renderer from "react-test-renderer"
import Nav from "../src/components/nav"

describe("Nav", () => {
	it("renders correctly", () => {
		const menuLinks = [{ name: "test", link: "/test" }]
		const tree = renderer.create(<Nav menuLinks={menuLinks} />).toJSON()
		expect(tree).toMatchSnapshot()
	})
})
