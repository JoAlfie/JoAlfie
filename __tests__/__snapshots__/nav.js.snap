// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<nav
  className="sitenav"
>
  <ul
    className="sitenav__list"
  >
    <li
      className="sitenav__item"
    >
      <a
        className="sitenav__link"
        href="/test"
      >
        test
      </a>
    </li>
  </ul>
</nav>
`;
